test/runtests.jl

```diff
@@ -113,19 +150,35 @@ const DATADIR = joinpath(@__DIR__, "data")
     end
 
     @test EDF._edf_repr(EDF._nearest_representable_edf_time_value(-0.0023405432)) == "-0.00234"
-    @test EDF._edf_repr(EDF._nearest_representable_edf_time_value(0.0023405432)) == "0.002340"
+    @test EDF._edf_repr(EDF._nearest_representable_edf_time_value(0.0023405432)) == "0.002341"
```

Comment:
this is because we're relying on `@sprintf` rounding instead of just truncating right?